test/pkg/environment/common/environment.go

```diff
@@ -113,10 +113,10 @@ func NewClient(ctx context.Context, config *rest.Config) client.Client {
 		node := o.(*corev1.Node)
 		return []string{strconv.FormatBool(node.Spec.Unschedulable)}
 	}))
-	lo.Must0(cache.IndexField(ctx, &corev1.Node{}, "spec.taints[*].karpenter.sh/disruption", func(o client.Object) []string {
+	lo.Must0(cache.IndexField(ctx, &corev1.Node{}, "spec.taints[*].karpenter.sh/disrupted", func(o client.Object) []string {
```

Comment:
This is fine to add here to fix things, but I'm wondering if we can decouple into a separate PR since this isn't related

Reply:
Since the testing fails and we already have it here - I think it's fine to leave in.